src/main/java/net/wurstclient/mixin/ClientCommonNetworkHandlerMixin.java

```diff
@@ -22,15 +22,15 @@
 public abstract class ClientCommonNetworkHandlerMixin
 	implements ClientCommonPacketListener
 {
-	@Inject(at = @At("HEAD"),
-		method = "sendPacket(Lnet/minecraft/network/packet/Packet;)V",
-		cancellable = true)
-	private void onSendPacket(Packet<?> packet, CallbackInfo ci)
+	@Redirect(method = "sendPacket(Lnet/minecraft/network/packet/Packet;)V",

```

Comment:
I want to try and avoid Redirect mixins as much as possible, since they can cause compatibility issues with other mods and hacks. Maybe there is a way to do this using one of the [MixinExtras](https://github.com/LlamaLad7/MixinExtras/wiki) mixins instead?

Reply:
It looks like [WrapOperation](https://github.com/LlamaLad7/MixinExtras/wiki/WrapOperation) works here. I'll use that instead.